.eslintrc.js

```diff
@@ -7,4 +7,10 @@ module.exports = {
       },
     },
   },
+  rules: {
+    '@typescript-eslint/camelcase': 'off',
+    'new-cap': 'off',
+    'import/no-unresolved': 'off', // False negatives
+    'import/named': 'off', // False negatives
```

Comment:
What did you need to disable this rule?

Reply:
Looks like eslint cannot resolve types definitions for several modules